cvat/apps/auto_annotation/model_manager.py

```diff
@@ -151,6 +153,13 @@ def save_file_as_tmp(data):
         labelmap_file = save_file_as_tmp(labelmap_file)
         interpretation_file = save_file_as_tmp(interpretation_file)
 
+    restricted = not has_admin_role(owner)
```

Comment:
In case update of the dl model we have None here as the owner. Will the code will work correctly? I believe if an admin user updates all files restricted will be False here. Could you please check?

@azhavoro , could you please remind us why `update_model(request, mid)` pass the owner as None? Should we change the owner if the user changes the DL model files?

Reply:
In case of update model it doesn't matter what passed as `owner` arg, it may be `request.user`. Now only owner or admin can update model `rules.add_perm('auto_annotation.model.update', (has_admin_role | is_model_owner) & ~is_primary_model)` and I think it's not good to change owner for updated model, in this case user just 'lose' his 
own model (user can see only owned models excluding shared).